source/quick_start_guide/index.html.md.erb

```diff
@@ -78,8 +77,7 @@ with our API and make a test API call.
 ![](/images/DescribeAPIKey+image2.png)
 <br /><br />Your API key will be shown for you to copy:<br /><br /> ![](/images/NewKeygenerate+image+3.png)
 
-[Read the __Security__ section](/security/#security) for more information on how
-to keep your API key safe.
+Make sure you [keep your API keys safe][https://docs.payments.service.gov.uk/security/#securing-your-developer-keys).
```

Comment:
make this a relative link

Reply:
Also fixed in https://github.com/alphagov/pay-tech-docs/pull/357/commits/31890ba88a10caa6ebc6f858308be8c7f81be604.